guides/discord-guide.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: 'How to use Novu to send notifications to a channel in a Discord server'
+description: 'Learn to send Discord notifications swiftly with Novu'
+icon: 'circle-5'
+---
+
+# Introduction
+
+In this guide, you'll learn how to use Novu to send notifications to any channel in a Discord server. But before coding anything up, we just need to go over a couple of setup steps.
+The corresponding docs for this guide are available on our [docs](https://docs.novu.co/channels-and-providers/chat/discord).
+<Note>The entire code for this app is available on our [Github](https://github.com/novuhq/discord-chat-app).</Note>
+
+So let's begin!
+
+# Set up Discord
+Setting up Discord is fairly straightforward. You just need the webhookUrl. It is pretty simple and can be done in the following easy steps:
+1. Go to the channel you want to add the webhook to (you need to be an admin of the discord server).
+<Frame caption="Go to the channel on Discord"><img src="/images/discord-guide/channel.png" /> </Frame>
+2. Right-click the channel and select “Edit Channel”.
+<Frame caption="Right click and select 'Edit'"><img src="/images/discord-guide/edit.png" /> </Frame>
+3. Select Integrations -> Webhooks -> Create Webhook
+<Frame caption="Create webhook from the Integrations menu"><img src="/images/discord-guide/integrations.png" /> </Frame>
+4. Edit the Bot name to your liking, copy the webhook URL and store it somewhere. We'll need it in the future.
+<Frame caption="Copy the webhookUrl and keep it somewhere"><img src="/images/discord-guide/webhook.png" /> </Frame>
+
+# Set up Novu
+In this part, we'll be setting up a workflow that will be triggered when we send a notification. Workflows are like blueprints and hold the entire flow of notifications. You can read more about them on our [docs](). To set up a notification workflow for our app, follow these steps:
```

Comment:
=> we'll set up

Reply:
In the part where you send you can read more about them on our docs..there is no link to that part of the docs. Please can you include it?